tests/uat/uat_sanity_suite_test.go

```diff
@@ -201,3 +201,17 @@ func TestVirtualDevice2PortCreateConnection_PNFV(t *testing.T) {
 	output := terraform.Output(t, terraformOptions, "port_connection_id")
 	assert.NotNil(t, output)
 }
+
+func TestVirtualDevice2AWSCreateConnection_DIGP(t *testing.T) {
```

Comment:
Which user is the test for? DIGP stands for diginpanthers and is our prod user. I know this is for UAT so make sure the suffix matches the user it's for.

Additionally, make sure the names of the resources you create have that suffix as well. Necessary for our test sweeping.

Reply:
I am testing it for PFCR User, updated the PR [here](https://github.com/equinix/terraform-equinix-fabric/pull/121/commits/e7f242eafd3735f93fed46875d5de72a39ee28ed)